### PR TITLE
Feat(ProspectInvitationPage) adding the invitation page with endpoint and db

### PR DESCRIPTION
**Pull request by dejanvasic85**

### CODE REVIEW

File: client/src/pages/InvitationPage.jsx [hunk 1]
```diff
@@ -1,51 +1,175 @@
...
+        [field]: true
       }
+    }
+
+    this.setState({ formData });
+  }
+
+  showValidation = (validation, field) => {
+    return validation[field] && this.state.formData.touched[field] === true
+  }
```
Comment (FastNinja) [inline client/src/pages/InvitationPage.jsx:84]: I swear I saw the same code :) in Agent page...

File: server/routes/prospects.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  }
+
+  const prospect = req.body;
+  prospect.status = 'New';
+
+  Prospect.create(prospect).then(p => {
+    res.status(201).json(p);
+  }).catch(err => {
+    res.status(500).json({ err });
+  });
```
Comment (FastNinja) [inline server/routes/prospects.js:20]: I wonder if it possible to make a global error middleware that will catch any exception or rejected promises.. and will return 500 along with error descripotion

**Review by FastNinja — APPROVED** [review]

File: client/src/pages/InvitationPage.jsx (hunk 1 above)
Comment (dejanvasic85) [reply client/src/pages/InvitationPage.jsx:84]: Yes. Duplicated code! We'll optimise and reuse later.

File: server/routes/prospects.js (hunk 2 above)
Comment (dejanvasic85) [reply server/routes/prospects.js:20]: This would be good

**Pull request merged**